### DISCUSSION TITLE
HotFix not detecting AddOn setups

### DISCUSSION THREAD

**Discussion by kalyani1217** [wixtoolset/issues#8612] [Questions]
I have created an AddOn setup for a base setup by setting the RelatedBundle action to AddOn. and also created a patch for the AddOn setup. However, only the base product is being detected in the DetectRelatedBundle action.

Why is the add-on related bundle ID not being detected during HF installation?

**Comment by bevanweiss** [comment]
So you have a base Bundle A, and then you have a patch Bundle B which has RelatedBundle=A.
What is your action configured to for the Patch Bundle B in it's RelatedBundle element (https://wixtoolset.org/docs/v3/xsd/wix/relatedbundle/)

So you now have two bundles:
A, and B

You now have a HotFix.  Which Bundle does this reference? A / B / BOTH A&B
If it's a HotFix, I would have assumed it only applies against your patched 'version', and hence would ONLY have a RelatedBundle=B
